utils/build.ps1

```diff
@@ -261,6 +348,20 @@ $LibraryRoot = "$ImageRoot\Library"
 $HostArch.ToolchainInstallRoot = "$(Get-InstallDir $HostArch)\Toolchains\$ProductVersion+Asserts"
 
 # Resolve the architectures received as argument
+$AndroidSDKArchs = @($AndroidSDKs | ForEach-Object {
+  switch ($_) {
+    "aarch64" { $AndroidARM64 }
+    "armv7" { $AndroidARMv7 }
+    "i686" { $AndroidX86 }
+    "x86_64" { $AndroidX64 }
+    default { throw "Unknown architecture $_" }
+  }
+})
+if ($Android) {
+  if ($HostArch -ne $ArchX64) {
```

Comment:
Ugh, `HostArch` is really `BuildArch`, this is really confusing currently.

Reply:
Hmm, how so?

Reply:
I was mistaken. HostArch is the toolchain host, there is `BuildArch` which is the build.